common-content/en/module/piscine/practice-break-down/index.md

```diff
@@ -15,7 +15,38 @@ In team projects, it's important that we break large tasks down into small tasks
 
 It's also important that we can coordinate across our team. This requires having shared understanding of who will do what, and how the work we do will interact.
 
-We will practice this together on the first two requirements of the project.
+Finally, it's important that we arrange tasks so your team isn't blocking yourselves, so you'll want to find ways of working on tasks in parallel.
+
+We will practice this together on the two of the requirements of the project:
+
+- Selecting a user must display the agenda for the relevant user (see manual testing below)
+- Submitting the form adds a new topic to revise for the relevant user only
+
+To complete these requirements we'd need to build most of the project! So we'll focus on simplified versions of these requirements:
+
+- When the page loads, display one revision date for one topic from in User 1's stored agenda
+- When clicking a button, store _hard-coded_ data for User 1's agenda
+
+To complete the full requirements, you can build on the tasks we decide on today.
+
+{{<note type="exercise" title="Breaking down tasks">}}
+
+1. Set a timer for {{<timer>}}5{{</timer>}}
+2. Individually, write down all the tasks that would be needed to complete the simplified requirements above
+3. After the timer is up, go around the group and discuss the tasks you came up with. One of the volunteers should write the tasks on a whiteboard
+4. Volunteers, discuss any that you think might be missing or that might need breaking down further
+
+{{</note>}}
+
+Some of these tasks _block_ other tasks: one task must be completed before the other can start.
```

Comment:
I realise this probably falls under "what tasks are there", but feel it's worth explicitly calling out?

```suggestion
Some of these tasks _block_ other tasks: one task must be completed before the other can start.

If different people are going to do some of these tasks, it's possible that _deciding_ or _agreeing_ some things in advance could enable those tasks to be worked on in parallel.
```

Reply:
Remember that there are activities in the backlog that talks about all this. Might be useful to use some of that?

Reply:
Yeah thanks. I originally had this framed as dependencies instead of blockers - as I thought the dependencies framing raises the idea of collaboration - but backed out as I thought they'd be more familiar with the blockers concept.

> there are activities in the backlog that talks about all this

Sorry if this should be obvious, but do you mean [this one](https://github.com/CodeYourFuture/The-Piscine/issues/6)?

Reply:
I've gone ahead and pushed a commit that reworks this exercise to be framed around the refining tickets coursework linked above.

I like this a lot more - it captures more of the workflow that they'll need to do.